tests/suites/test_suite_pk.data

```diff
@@ -708,38 +708,10 @@ PSA attributes for pk: RSA v15 pair DECRYPT
 depends_on:PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY:PSA_WANT_ALG_RSA_PKCS1V15_SIGN
 pk_get_psa_attributes:MBEDTLS_PK_RSA:FROM_PAIR:PSA_KEY_USAGE_DECRYPT:1:PSA_ALG_RSA_PKCS1V15_CRYPT
 
-PSA attributes for pk: RSA v21 SHA-256 pair DECRYPT
-depends_on:PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY:PSA_WANT_ALG_RSA_PSS:PSA_WANT_ALG_SHA_256
-pk_rsa_v21_get_psa_attributes:MBEDTLS_MD_SHA256:FROM_PAIR:PSA_KEY_USAGE_DECRYPT:1:PSA_ALG_RSA_PKCS1V15_CRYPT
-
-PSA attributes for pk: RSA v21 SHA-512 pair DECRYPT
-depends_on:PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY:PSA_WANT_ALG_RSA_PSS:PSA_WANT_ALG_SHA_512
-pk_rsa_v21_get_psa_attributes:MBEDTLS_MD_SHA512:FROM_PAIR:PSA_KEY_USAGE_DECRYPT:1:PSA_ALG_RSA_PKCS1V15_CRYPT
-
-PSA attributes for pk: RSA v15 pair->public ENCRYPT
```

Comment:
This v15 test case was probably delete by mistake with the surrounding v21 cases.